data/abilities/leap.json

```diff
@@ -1,9 +1,11 @@
 {
     "name": "Leap",
+    "description": "Moves yourself to another location, that could've been out of reach.",
```

Comment:
"Moves the golem" vs "Moves yourself" (the other descriptions are third-person and reference the golem)

What is our thoughts on contractions in these descriptions? They are discouraged in formal writing, but I do not know how "chatty" we want to get. I would lean toward formal, but that's just my default when writing stuff for others to consume.

The comma is not necessary here, but it is not incorrect to include it. No strong preference here, but I would lean toward removing it to help keep descriptions shorter (though one character does not matter here).

Reply:
Yeah I started to catch myself as I was going along, but looks like some slipped through the cracks. Good catch.

As for contractions, I don't think we need it. To me the descriptions should likely read as though you're reading from a technical manual.

To the later's point perhaps we should use more words, but possibly highlight the specifically useful ones. That's a down the road thing though, so in this case I'd say go super formal. We can come up with jargon in a later update.